Validation/HGCalValidation/python/HGCalValidator_cfi.py

```diff
@@ -18,11 +18,14 @@
     #2dlayerclusters, pfclusters, multiclusters
     label_lcl = layerClusterCaloParticleAssociation.label_lc,
     label_mcl = cms.VInputTag(
+      cms.InputTag("ticlMultiClustersFromTrackstersTrkEM"),
```

Comment:
ticlMultiClustersFromTrackstersTrkEM was already present, two lines below

Reply:
thanks @mariadalfonso , there was another PR in the mean time that added it and git did not find this as a merge conflict. I will fix it